RELEASE/scripts/autoscend/iotms/mr2017.ash

```diff
@@ -5,7 +5,7 @@ script "mr2017.ash"
 boolean mummifyFamiliar(familiar fam, string bonus)
 {
 	bonus = to_lower_case(bonus);
-	if(auto_my_path() == "Pocket Familiars")
+	if(auto_my_path() == "Pocket Familiars" || auto_my_path() == "Dark Gyffte" || my_path() == "License to Adventure")
```

Comment:
```suggestion
	if(!pathAllowsFamiliar())
```

Reply:
we got a function we keep up to date that checks all possible familiar forbidding paths

Reply:
Did you test and find that mummifyfamiliar is causing errors as a vampyre or license to adventure?
This was related to an issue Asmodais reported in the discord this morning, he tested after putting my change in and it ran fine after he put the changes in. Not sure why it wouldnt have worked.